.yamato/_projectcontext.yml

```diff
@@ -417,6 +417,8 @@ all_package_ci_project_2019.4:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  upm-ci package izon -t
       -  upm-ci package izon -d
+    triggers:
+        expression: pull_request.target eq "7.x.x/release" AND NOT pull_request.draft AND NOT pull_request.push.changes.all match ["**/*.md", "doc/**/*", "**/Documentation*/**/*", ".github/**/*", "Tools/**/*"]
```

Comment:
Aren't we supposed to run the job even if the PR contains only Documentation? Looks like the expression is different from the one in the original PR https://github.com/Unity-Technologies/Graphics/pull/3652/files

Reply:
ah good catch, thank you